recipes/wfdb/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "wfdb" %}
+{% set version = "1.3.5" %}
+{% set sha256 = "a8964ac7c4f0bb70a571bd4236e5324a6297a2a6b6cfbf62f057811295668cfb" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```

Comment:
The `--single-version-externally-managed` flag to `setup.py` means that you only need packages in the list `build` requirements that are either imported by `setup.py` or are in `setup_requires`. I think you only need `python` and `setuptools` here.

Reply:
Most build dependencies were removed, I only kept python and setuptool.